app/adapter/graphql/resolver/url_test.go

```diff
@@ -0,0 +1,59 @@
+package resolver
```

Comment:
I am thinking of doing integration tests for the GraphQL APIs instead. Send HTTP request, and get response. Unit tests seem to be good for use cases. What are your thoughts?

(https://github.com/byliuyang/short/blob/master/backend/app/adapter/graphql/resolver/query.go) instead. What are your thoughts?

Reply:
Integration tests for integration, unit tests for units. But I agree, maybe it's okay use int. tests in this context